assets/js/events.json

```diff
@@ -0,0 +1,52 @@
+{
+    "event-admin": [
+        {
+            "video name": null,
+            "event image":"code ada team bonding.png",
+            "name": "Code Ada Team Matching Social",
+            "date": "10/11/2022",
+            "slide link": [],
+            "description":"Looking for teammates for Code Ada? Want to brainstorm some project ideas before the hackathon? Come to our second Team Matching Social on October 11th from 5-6pm! We’ll go over some possible project ideas, tips and tricks for the day of, and we’ll be playing some games! We will also have snacks! Stop by to relax, play, and meet new people!"
+        },
+        {
+            "video name": "20220922-git-workshop.mp4",
+            "event image": [],
+            "name": "Git Gud @ Git",
+            "date": "09/22/2022",
+            "slide link": "https://docs.google.com/presentation/d/1ypeJc3I3eklbVBF5hmdLZINVg3UiHA8l2DU2gVdKQXM/edit?usp=sharing",
+            "description": "Want to learn more about using GitHub? Interesed in getting practice with version control and Git? Come Git Gud @ Git at our GitHub workshop! No experience is necessary, and we welcome all ages, majors, and backgrounds."
+        },
+        {
+            "video name":[],
```

Comment:
Choose to make a nonexistent video either an empty array or null for consistency